app/src/common/shared/org/mozilla/vrbrowser/ui/widgets/Windows.java

```diff
@@ -249,6 +249,11 @@ public WindowWidget addWindow() {
         focusWindow(newWindow);
         updateCurvedMode(true);
         updateViews();
+
+        // We are only interested in general windows opened.
+        if (!isInPrivateMode()) {
```

Comment:
I guess this has been a conscious decision but we are sending open window events for private windows too in the old telemetry.

Reply:
This is interesting. @daoshengmu was this clearly called out and discussed with Data Stewards?

Reply:
I got this request from here, https://github.com/MozillaReality/FirefoxReality/issues/2347#issuecomment-558807732

@jvonitter Can you confirm it again? If so, we will only collect `open new window` events for non-private mode.